src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -57,6 +59,16 @@ protected function enqueueResources(Group $pools, Database $dbForPlatform, calla
 
             $delay = $scheduledAt->getTimestamp() - (new \DateTime())->getTimestamp();
 
+            $project = $schedule['project'];
```

Comment:
What is the difference between this file and the functions scheduler?!

Reply:
I think one is for delayed execution from console and another is for scheduled executions, I don't know exactly